### ISSUE TITLE
Debugger refuses to connect

### ISSUE THREAD

**Issue by JstnPwll** [monaca/monaca-cli#128]
It's not clear whether this is an issue with the cli or the Android app, but I'd at least appreciate knowing where to look next. Here's what I've done:

- I have installed the latest monaca cli on my PC (2.7.0)
- I have installed the latest monaca debugger Android app on my phone (7.0.2)
- I have completely disabled firewall on my PC
- I connected my PC and phone to the same WiFi access point
- As soon as both connect, the debugger sees the PC and asks if I want to pair
- Every time, pairing is unsuccessful. No information on the cli
- I have tried restarting the cli using different ports with no success
- I know my phone can access my PC, because I can navigate to a local webserver on my PC via my phone's browser
- In other words, only Monaca is having this problem
- This also happens when I connect directly to my phone's WiFi hotspot, ruling out the router

Is there anywhere I can look for help debugging this issue? I'm getting no output whatsoever from the cli, and the only info I get from the app is "Pairing unsuccessful".
